src/orion/benchmark/assessment/averagerank.py

```diff
@@ -39,77 +36,10 @@ def analysis(self, task, experiments):
             A list of (task_index, experiment), where task_index is the index of task to run for
             this assessment, and experiment is an instance of `orion.core.worker.experiment`.
         """
-        task_algorithm_exp = defaultdict(list)
+        algorithm_groups = defaultdict(list)
 
-        for task_index, exp in experiments:
+        for _, exp in experiments:
```

Comment:
Since we now do not add `task name` into the plot and also not require the `task index`, let me simplify the input for the `analysis` call.

Reply:
We may want to make the title of the plot configurable. I avoided it for now but it's probably to add this. If we have many plots having the name of the task will help understand what is what.

Reply:
okay. let's keep it first and see what can do later.